themes/bootstrap3/templates/combined/stack-grid.phtml

```diff
@@ -0,0 +1,37 @@
+<div class="combined-search-container grid"
+  style="grid-template-columns: repeat(<?=$this->escapeHtmlAttr($columns)?>, 1fr)"
+>
+  <?php $columnedParams = []; ?>
+  <?php $columnIndex = 0; ?>
+  <?php foreach ($this->combinedResults as $searchClassId => $currentSearch): ?>
+    <?php
+      if (
+        (!isset($currentSearch['ajax']) || !$currentSearch['ajax'])                    // AJAX column
+        && (isset($currentSearch['hide_if_empty']) && $currentSearch['hide_if_empty']) // set to hide when empty
+        && $currentSearch['view']->results->getResultTotal() == 0                      // and empty
+      ) {
+        continue;
+      }
+      $colParams = [
+        'searchClassId' => $searchClassId,
+        'currentSearch' => $currentSearch,
+        // Enable cart if appropriate:
+        'showCartControls' => $this->supportsCartOptions[$columnIndex] && $this->showCartControls,
+        // Enable bulk options if appropriate:
+        'showBulkOptions' => $this->supportsCartOptions[$columnIndex] && $this->showBulkOptions,
+        // Other params
+        'domId' => $currentSearch['domId'],
+      ];
+      if (isset($currentSearch['ajax'])) {
+        $colParams['ajax'] = $currentSearch['ajax'];
+      }
+    ?>
+    <?php // This empty div prevents margin collapse, as the otherwise unneeded div.combined-column does elsewhere. ?>
+    <div>
+      <div id="<?=$this->escapeHtmlAttr($colParams['domId'])?>" class="combined-list">
+        <?php $templateSuffix = (isset($colParams['ajax']) && $colParams['ajax']) ? 'ajax' : 'list'; ?>
```

Comment:
`isset($colParams['ajax']) && $colParams['ajax']` could be simplified to `$colParams['ajax'] ?? false` for improved readability. (php-cs-fixer is supposed to spot and correct this kind of scenario, but it doesn't always work).